io.openems.edge.heater.api/src/io/openems/edge/heater/api/Heater.java

```diff
@@ -0,0 +1,419 @@
+package io.openems.edge.heater.api;
+
+import io.openems.common.channel.AccessMode;
+import io.openems.common.channel.Level;
+import io.openems.common.channel.PersistencePriority;
+import io.openems.common.channel.Unit;
+import io.openems.common.types.OpenemsType;
+import io.openems.edge.common.channel.Channel;
+import io.openems.edge.common.channel.Doc;
+import io.openems.edge.common.channel.IntegerReadChannel;
+import io.openems.edge.common.channel.LongReadChannel;
+import io.openems.edge.common.channel.StateChannel;
+import io.openems.edge.common.channel.value.Value;
+import io.openems.edge.common.component.OpenemsComponent;
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * A generalized nature for a heater. Provides channels that should be available
+ * on all heaters, allowing a vendor agnostic implementation. Vendor specific
+ * interfaces should extend this interface.
+ */
+@ProviderType
+public interface Heater extends OpenemsComponent {
+
+    public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
+
+	/**
+	 * Temperature value of the outgoing water flow.
+	 *
+	 * <ul>
+	 * <li>Interface: Heater
+	 * <li>Type: Integer
+	 * <li>Unit: Decidegree Celsius
+	 * </ul>
+	 */
+	FLOW_TEMPERATURE(Doc.of(OpenemsType.INTEGER) //
+		.unit(Unit.DEZIDEGREE_CELSIUS) //
+		.persistencePriority(PersistencePriority.MEDIUM) //
+		.accessMode(AccessMode.READ_ONLY)), //
+
+	/**
+	 * Temperature value of the water return flow.
+	 *
+	 * <ul>
+	 * <li>Interface: Heater
+	 * <li>Type: Integer
+	 * <li>Unit: Decidegree Celsius
+	 * </ul>
+	 */
+	RETURN_TEMPERATURE(Doc.of(OpenemsType.INTEGER) //
+		.unit(Unit.DEZIDEGREE_CELSIUS) //
+		.persistencePriority(PersistencePriority.MEDIUM) //
+		.accessMode(AccessMode.READ_ONLY)), //
+
+	/**
+	 * Output Heating power.
+	 *
+	 * <ul>
+	 * <li>Interface: Heater
+	 * <li>Type: Integer
+	 * <li>Unit: Watt
+	 * </ul>
+	 */
+	HEATING_POWER(Doc.of(OpenemsType.INTEGER) //
+		.unit(Unit.WATT) //
+		.persistencePriority(PersistencePriority.MEDIUM) //
+		.accessMode(AccessMode.READ_ONLY)), //
+
+	/**
+	 * Output Heating energy.
+	 *
+	 * <ul>
+	 * <li>Interface: Heater
+	 * <li>Type: Integer
+	 * <li>Unit: KilowattHours
+	 * </ul>
+	 */
+	HEATING_ENERGY(Doc.of(OpenemsType.LONG) //
+		.unit(Unit.KILOWATT) //
```

Comment:
Found a typo: Unit should be KilowattHours

Reply:
Thanks, fixed it.